src/python/grpcio_health_checking/grpc/health/v1alpha/health.py

```diff
@@ -0,0 +1,73 @@
+import abc
+import enum
+from grpc.health.v1alpha import health_pb2
+
+
+@enum.unique
+class HealthStatus(enum.Enum):
+  UNKNOWN = health_pb2.HealthCheckResponse.UNKNOWN
+  SERVING = health_pb2.HealthCheckResponse.SERVING
+  NOT_SERVING = health_pb2.HealthCheckResponse.NOT_SERVING
+
+
+class _HealthServicer(health_pb2.EarlyAdopterHealthServicer):
+
+  def __init__(self):
+    self._server_status = {}
```

Comment:
This field appears to be accessed from multiple threads - how about protecting it with a threading.Lock?